src/v/cluster/tx_gateway_frontend.cc

```diff
@@ -1810,9 +1738,35 @@ ss::future<tx_errc> tx_gateway_frontend::reabort_tm_tx(
         auto grs = co_await when_all_succeed(gfs.begin(), gfs.end());
         auto ok = true;
         for (const auto& r : prs) {
+            if (r.ec != tx_errc::none) {
+                vlog(
+                  txlog.trace,
+                  "remote aborting tx:{} etag:{} pid:{} tx_seq:{} term:{} "
+                  "failed with {}",
+                  tx.id,
+                  tx.etag,
+                  tx.pid,
+                  tx.tx_seq,
+                  expected_term,
+                  tx.status,
+                  r.ec);
+            }
             ok = ok && (r.ec == tx_errc::none);
         }
         for (const auto& r : grs) {
+            if (r.ec != tx_errc::none) {
+                vlog(
+                  txlog.trace,
+                  "remote aborting tx:{} etag:{} pid:{} tx_seq:{} term:{} "
+                  "failed with {}",
+                  tx.id,
+                  tx.etag,
+                  tx.pid,
+                  tx.tx_seq,
+                  expected_term,
+                  tx.status,
+                  r.ec);
```

Comment:
here also, mismatched number of place holders

Reply:
fixed